ui/admin/app/components/form/auth-method/ldap/index.hbs

```diff
@@ -212,6 +212,19 @@
     </Hds::Form::TextInput::Field>
   {{/if}}
 
+  {{#unless @model.isNew}}
+    <Hds::Form::TextInput::Field
```

Comment:
I don't have context for this but do we need to show the `HMAC`? Is it useful for the user in this case, especially considering how large the form already is? 

I know we show it in the OIDC form and I think vault cred store but we stopped showing it in the newer forms we've built since then.

If we do display it though, these fields should be [readonly](https://helios.hashicorp.design/components/form/text-input#readonly-disabled-and-hidden-fields) instead of just disabled.

Reply:
@ZedLi This is a question I presented to Randy, whose recommendation was to show it.  Fast forward, I wanted to display both the `client_certificate,` `client_certificate_key`, `bind_dn`, and `bind_password` and do them as we do `secret_access_key`.  @DhariniJeeva and @joshuaogle what do you think?

<img width="1426" alt="Screenshot 2023-07-25 at 16 46 48" src="https://github.com/hashicorp/boundary-ui/assets/24277002/f744bb4c-0099-4837-a7f3-c503bb38b39f">

Reply:
Good point. The user doesn't need to see the HMAC info. I'm ok with getting rid of those from the form. We should also do this in OIDC for consistency in auth method forms. @dvonanderson, can you please add a ticket in the backlog to remove the HMAC from OIDC if we decide not to show the field?

Reply:
That sounds great to me too, and I like the idea of adopting that in OIDC. Thanks!